src/ui/public/index_patterns/__tests__/_index_pattern.js

```diff
@@ -12,8 +12,11 @@ import IndexPatternsMapperProvider from 'ui/index_patterns/_mapper';
 import UtilsMappingSetupProvider from 'ui/utils/mapping_setup';
```

Comment:
Now that the stubbed logstash index pattern fields have aggregatable and searchable properties, the call to refresh the fields is not happening, which means that the stubbed version of `mapper.getFieldsForIndexPattern()` is not being used and the strategic points where bluebird promises were injected are no longer correct. Rather than with the fragile practice of only replacing the promises we need, the `no_disgest_promises` test until just replaces our `Promise` angular module with `Bluebird` promises, so that they resolve like normal and don't require calls to `$rootScope.$apply()`.